keps/sig-network/563-dual-stack/README.md

```diff
@@ -1380,9 +1380,9 @@ This capability will move to beta when the following criteria have been met.
 This capability will move to stable when the following criteria have been met.
 
 * Support of at least one CNI plugin to provide multi-IP
-* e2e test successfully running on two platforms
+* e2e test successfully running on two platforms (Azure and kind)
 * testing ingress controller infrastructure with updated dual-stack services
-* dual-stack tests run as pre-submit blocking for PRs
+* dual-stack tests always run as pre-submit (non-blocking) for PRs
```

Comment:
We do have an open issue to support dual-stack clusters in CAPZ: https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/1266. After that point, we can move these tests off AKS-engine to CAPZ. We don't want k8s code merges to take an external dependency on AKS-engine, as CAPZ is the route forward.

@aramase will change this presubmit job (which already exists) to always run.